### DISCUSSION TITLE
Clips location (mp4) on volume?

### DISCUSSION THREAD

**Discussion by MrSiO** [blakeblackshear/frigate#9027] [Ask A Question] [answered]
Hi.
Simple question.

From HA Media and from Frigate UI I can see Clips are present and available for download. When downloaded the file starts with "clip-xxxx.mp4".

But for some reason I cannot find those clip files anywhere on my media folder...

Under Clips I only got snapshots images.
Under recording I got full MP4 but no clips.

(I'm running latest Beta on Docker)

Question is: where are those files stored?

Thx!

**Comment by NickM-27** [comment]
The files are stored nowhere, they are dynamically generated from the recording segments and combined and then downloaded.

**Reply by MrSiO** [reply]
What about the clips in HA's media?

Are those stored somewhere or also generated?

**Reply by NickM-27** [reply, marked as answer]
The clips in HA media are just streamed on the fly when you watch, they are not when combined into a single file

**Reply by MrSiO** [reply]
Thx for the info!
